BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemViewModel.swift

```diff
@@ -16,6 +15,7 @@ protocol EditLineItemViewModelDelegate: AnyObject {
 final class EditLineItemViewModel {
     weak var delegate: EditLineItemViewModelDelegate?
     private var lineItem: DigitalInvoice.LineItem
+    var itemsChanged: [LineItemAnalytics] = []
```

Comment:
[Clarification]: can we make this var private?

Reply:
I've made just the `set` private. I'm using it outside of this file.